file.c

```diff
@@ -6537,6 +6537,142 @@ const char ruby_null_device[] =
  *  read-only, which is reported as <code>0444</code>.
  *
  *  Various constants for the methods in File can be found in File::Constants.
+ *
+ *  == What's Here
+ *
+ *  First, what's elsewhere. \Class \File:
+ *
+ *  - Inherits from class IO -- in particular, methods for creating,
+ *    reading, and writing files.
+ *  - Includes module FileTest.
+ *
+ *  \Class \File provides methods for:
+ *
+ *  === Creating
+ *
+ *  - ::new:: Creates a new file at the given file path; returns the file.
+ *  - ::open:: Creates a new file at the given file path;
```

Comment:
`Same as ::new, but when given a block will yield the file to the block, and close the file upon exiting the block.` It also doesn't create files by default.